i18n/english.yml

```diff
@@ -1,371 +1,368 @@
-id: en-US
-name: English
```

Comment:
I somewhat prefer having `id` and `name` on the first few lines of the i18n files.

Maybe we should nest all of the alphabetized components under a `components` key.

Reply:
I'm using https://github.com/akfernun/yaml-sortkeys to sort everything and would rather not make exceptions like this.  FWIW, since every other first-level key is capitalized, these keys end up at the very bottom.

Reply:
@evansiroky, shall we just put underscores in front of id and name (`_id` and `_name`)?

Reply:
Prefixing with an underscore won't work because underscores have a higher char code than capital letters.